### PR TITLE
Hello-world 

### PR DESCRIPTION
**Pull request by misikirdelelew**
In this simple project, I have added
-index.html
-linter and its dependencies
-updated the readme file

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @MishDM, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



- [ ] What do you think about having a professional readme? It sounds good, right? You can have one for yourself. Kindly follow this [template](https://github.com/microverseinc/readme-template/blob/master/README.md). This will help you to write the GET STARTED section professionally.
- [ ] Lint, or a linter, is a static code analysis tool used to flag programming errors, bugs, stylistic errors and suspicious constructs and can help you auto-correct some things like indentation and make your code of general acceptance. Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to setup the linter.
- [ ] Kindly make sure you have submitted for review and you have been approved before you merge, I can see that your index.html file is in the main branch already whereas, it should appear alongside the style.css file here.
- [ ] Kindly create a `.gitignore` file
- [ ] Kindly create a test.md file and add it to the `.gitignore` file so that it will not be shown in the repository.
- [ ] When you successfully add the [linter](https://github.com/microverseinc/linters-config/tree/master/html-css), a node_modules folder will also be added, kindly add the node_modules to `.gitignore`. This is because it has a large size.


## Highlights
- Good commit messages ✔️ 
- Good use of git and github flow ✔️ 
- Project renders on UI ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+# .gitignore
 node_modules/
```
Comment (topeogunleye) [inline .gitignore]: - [ ] Great job adding test.md to your `.gitignore` file. However you haven't a created a `test.md` file as suggested by the previous reviewer. Please do so.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @MishDM, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @MishDM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good commit messages ✅ 
- Good use of Git and Github flow ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**